### ISSUE TITLE
allow verbatim buildOptions (eg for flags not yet supported by dub's buildOptions, eg -vcolumns)

### ISSUE THREAD

**Issue by timotheecour** [dlang/dub#1400]
`-vcolumns` is not in buildOptions (https://code.dlang.org/package-format?lang=json ); while we could add it there, I'd like a more general solution of allowing verbatim flags that would allow any flag to be added explicitly even if not yet supported by dub's buildOptions. This would be also useful for very recent flags, or experimental flags in a user's hacked dmd compiler, or obscure ldc options, etc.

eg usage (in sdl syntax):
```
buildOptions "debugInfo" "-vcolumns" "-custom_flag"
```
basically anything starting with "-" would be interpreted as a verbatim flag meant to be passed directly to compiler cmd line.

alternative syntax: use a different group, eg:
```
buildOptionsVerbatim "-vcolumns" "-custom_flag"
```
